### PR TITLE
itmedia.co.jp - Ad

### PR DESCRIPTION
**Pull request by SKEIDs**
### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [x] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [x] Filters maintenance.

### Enter the issue address

NO.1 - https://www.itmedia.co.jp/news/articles/2209/12/news183.html
NO.2 - https://www.itmedia.co.jp/business/articles/2209/13/news008.html

### Add your comment and screenshots
・Ad
1 - `itmedia.co.jp##div[id^="ITR"]`
2 - `itmedia.co.jp###cmsSubPR` https://github.com/AdguardTeam/AdguardFilters/pull/129596#discussion_r972239648

・Self-promo
https://github.com/AdguardTeam/AdguardFilters/pull/129596#discussion_r972223323
3.1 - `itmedia.co.jp###ITT`
3.2 - `itmedia.co.jp###RANKTXT`

<details><summary>screenshot - 1</summary>

</details><br/>

<details><summary>screenshot - 2 (website URL is NO.2)</summary>

</details><br/>

<details><summary>screenshot - 3</summary>

</details><br/>

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

**Comment by SKEIDs** [comment]
@Alex-302 
Should I recreate the this pull request?

File: JapaneseFilter/sections/specific.txt [hunk 1]
```diff
@@ -1998,10 +1995,23 @@ intensive911.com##div[id^="adx_"]
...
+/ad.js$domain=itmedia.co.jp
+||itmedia.co.jp^*/file_bit_article_bottom_ad.html
+||itmedia.co.jp/css/sa.js
+||itmedia.co.jp/css/sa_ini.js
 itmedia.co.jp###ATF
+itmedia.co.jp###ITN
+itmedia.co.jp###cmsBody > div.inner > div > div[style="text-align:center;margin:40px 0;"]
 itmedia.co.jp###cmsBodyMain > div:not([class]) > div[style="text-align:center;margin-bottom:20px;"]:only-child
 itmedia.co.jp##div[style^="font-family:Gill Sans;"][style*="font-size:12px;"][style*="line-height:12px;"]
+itmedia.co.jp##div[id^="ITR"]
```
Comment (Yuki2718) [inline JapaneseFilter/sections/specific.txt:2007]: Better `###ITR` + `itmedia.co.jp###globalHeader`

**Review by Yuki2718 — CHANGES_REQUESTED** [review]

File: JapaneseFilter/sections/specific.txt [hunk 2]
```diff
@@ -1998,10 +1995,23 @@ intensive911.com##div[id^="adx_"]
...
 quick-timez.com##.k1y-inline-ad
 ferret-plus.com##div[class*="-ad-banner"]
-itmedia.co.jp###ITN
+/ad.js$domain=itmedia.co.jp
+||itmedia.co.jp^*/file_bit_article_bottom_ad.html
+||itmedia.co.jp/css/sa.js
+||itmedia.co.jp/css/sa_ini.js
 itmedia.co.jp###ATF
+itmedia.co.jp###ITN
+itmedia.co.jp###cmsBody > div.inner > div > div[style="text-align:center;margin:40px 0;"]
```
Comment (Yuki2718) [inline JapaneseFilter/sections/specific.txt:2004]: On my end this doesn't hit. Do you see `#ITT`?

**Review by Yuki2718 — CHANGES_REQUESTED** [review]

File: JapaneseFilter/sections/specific.txt (hunk 1 above)
Comment (SKEIDs) [reply JapaneseFilter/sections/specific.txt:2007]: `###ITR`,`###ITR2` and `###ITRN` are there.
Should I write the rule separately?

File: JapaneseFilter/sections/specific.txt [hunk 3]
```diff
@@ -1998,10 +1995,23 @@ intensive911.com##div[id^="adx_"]
 clicccar.com##.ad-management
 quick-timez.com##.k1y-inline-ad
 ferret-plus.com##div[class*="-ad-banner"]
-itmedia.co.jp###ITN
+/ad.js$domain=itmedia.co.jp
+||itmedia.co.jp^*/file_bit_article_bottom_ad.html
+||itmedia.co.jp/css/sa.js
+||itmedia.co.jp/css/sa_ini.js
```
[the remark this answers is not in the document]
Comment (SKEIDs) [reply JapaneseFilter/sections/specific.txt:2001]: I just move the rule,so I am not sure if the rule is still work. 
Do you want to delete it?
These rules are added from this commit.
`/ad.js$domain=itmedia.co.jp`
https://github.com/AdguardTeam/AdguardFilters/commit/d6fe255a60bdfad29d7c232d0273fdf468bcba59
`||itmedia.co.jp^*/file_bit_article_bottom_ad.html`
https://github.com/AdguardTeam/AdguardFilters/commit/11e7ca1bc3228b73ed97000f6a217b2ed21772e7
`||itmedia.co.jp/css/sa.js` , `||itmedia.co.jp/css/sa_ini.js`
https://github.com/AdguardTeam/AdguardFilters/commit/fe6a7cbd00c2ef44fe82f288f919eb716aaea0f0

File: JapaneseFilter/sections/specific.txt (hunk 2 above)
Comment (SKEIDs) [reply JapaneseFilter/sections/specific.txt:2004]: I just move the rule,so I am not sure if the rule is still work.
Do you want to delete it?
These rules are added from this commit.
`itmedia.co.jp###ITN`
https://github.com/AdguardTeam/AdguardFilters/commit/c8c9c7028b1ab9d6b97a9d5768f5c1bfbaf87fb8
`itmedia.co.jp###cmsBody > div.inner > div > div[style="text-align:center;margin:40px 0;"]`
https://github.com/AdguardTeam/AdguardFilters/commit/ff64356b2035f6ea4c9a4f56be366581f6d6678a

Comment (SKEIDs) [reply JapaneseFilter/sections/specific.txt:2004]: I can see `#ITT` in issue address
<details><summary>screenshot</summary>



</details><br/>

File: JapaneseFilter/sections/specific.txt (hunk 1 above)
Comment (Yuki2718) [reply JapaneseFilter/sections/specific.txt:2007]: Hm, I missed ITR2. In that case maybe `##div[id^="ITR"]` is maybe okay. Butno need for `###ITRN` as covered by `###globalHeader`.

File: JapaneseFilter/sections/specific.txt (hunk 2 above)
Comment (Yuki2718) [reply JapaneseFilter/sections/specific.txt:2004]: Oh, I mistook. I still se the requests on the issue URL so no need to remove.

Comment (Yuki2718) [reply JapaneseFilter/sections/specific.txt:2004]: If `#ITT` covers the target, it'll be better.

File: JapaneseFilter/sections/specific.txt (hunk 1 above)
Comment (SKEIDs) [reply JapaneseFilter/sections/specific.txt:2007]: I see.

File: JapaneseFilter/sections/specific.txt (hunk 2 above)
Comment (SKEIDs) [reply JapaneseFilter/sections/specific.txt:2004]: `#ITT` is PR article.
Should I make a new rule as self-promo?

File: JapaneseFilter/sections/specific.txt (hunk 3 above)
[the remark this answers is not in the document]
Comment (Yuki2718) [reply JapaneseFilter/sections/specific.txt:2001]: Oh, I mistook. I still see the requests on the issue URL so no need to remove.

File: JapaneseFilter/sections/specific.txt (hunk 2 above)
Comment (Yuki2718) [reply JapaneseFilter/sections/specific.txt:2004]: You're right, sorry it seems I can't do anything right today. Apparently `###cmsBody > div.inner > div > div[style="text-align:center;margin:40px 0;"]` is obsolete now so can be removed.

Comment (Yuki2718) [reply JapaneseFilter/sections/specific.txt:2004]: On my end those PR leads to unrelated third-party sites. If this is true to you too, they can be addressed in JPF.

File: JapaneseFilter/sections/specific.txt (hunk 3 above)
[the remark this answers is not in the document]
Comment (SKEIDs) [reply JapaneseFilter/sections/specific.txt:2001]: I see.

File: JapaneseFilter/sections/specific.txt (hunk 2 above)
Comment (SKEIDs) [reply JapaneseFilter/sections/specific.txt:2004]: I see. I will add rule of self-promo and remove `###cmsBody > div.inner > div > div[style="text-align:center;margin:40px 0;"]` . Thank you for review amd advice.

File: AnnoyancesFilter/Other/sections/self-promo.txt [hunk 4]
```diff
@@ -1473,6 +1473,7 @@ theguardian.com##.contributions__adblock
 ! SECTION: Self-promo - Regular rules
 !
 detail.chiebukuro.yahoo.co.jp###v_bnr
+itmedia.co.jp##.colBoxITT
```
Comment (Yuki2718) [inline AnnoyancesFilter/Other/sections/self-promo.txt:1476]: Are they linked to their own group sites? I meant if they lead to third-party sites e.g. `https://www.pref.yamaguchi.lg.jp/soshiki/32/101159.html`, you can add `itmedia.co.jp##ITT` to Japanese filter.

Comment (SKEIDs) [reply AnnoyancesFilter/Other/sections/self-promo.txt:1476]: The two at the top of the site go to third-party site.
<details><summary>screenshot - 1</summary>



</details><br/>

But `##.colBoxITT` on the sidebar of the site go to own group site.

<details><summary>screenshot - 2</summary>



</details><br/>

Comment (Yuki2718) [reply AnnoyancesFilter/Other/sections/self-promo.txt:1476]: Maybe
`###ITT` => JPF
`###RANKTXT` => Annoyances

Comment (Yuki2718) [reply AnnoyancesFilter/Other/sections/self-promo.txt:1476]: Please also add `###cmsSubPR` to JPF seen in `https://www.itmedia.co.jp/business/articles/2209/13/news008.html`

Comment (SKEIDs) [reply AnnoyancesFilter/Other/sections/self-promo.txt:1476]: I see. I will change the rule

**Review by Yuki2718 — APPROVED** [review]

**Comment by Yuki2718** [comment]
Thank you, sorry for my confusion.

**Comment by SKEIDs** [comment]
> Thank you, sorry for my confusion.

It was really helpful. many thanks for all of your help.

**Review by zloyden — APPROVED** [review]

**Pull request merged**